### ISSUE TITLE
Accessibility: In tag picker list, Screen reader users are not informed about the remove button under "Picker" selected list item.

### ISSUE THREAD

**Issue by v-kusudh** [microsoft/fluentui#13018]
The bug is reproduce in office fabric official website.
   https://developer.microsoft.com/en-us/fluentui#/controls/web/pickers
![image](https://user-images.githubusercontent.com/64893219/81153426-8957e680-8fa0-11ea-90be-20c1bfde2f67.png)

Browser & Version:
Chrome: Version 80.0.3987.163 (Official Build) (64-bit)

Chromium edge: Version 81.0.416.53 (Official build) beta (64-bit)

Screen reader: NVDA and Narrator

OS Details:

Microsoft Windows 10 Enterprise

Version 1909 (OS Build 18363.657)

Actual Result:​
Screen reader users are not informed about the remove button under "Picker" selected list item.

Expected Result:​
Screen reader users should be informed about the remove button under "Picker" selected list item. 
When the focus lands on the selected list item the screen reader should not read as level 1 of 1 instead it should read level 1 of 2 so the user can navigate to the "remove" button using the arrow key.

User Impact:​
Screen reader users will not know how to remove the list item if the screen reader doesn't announce about the another element in the group.

**Comment by aneeshack4** [comment]
@v-kusudh This doesn't repro on Chrome or Edge with Narrator - this is the output get, you can see the remove button is being announced.
![image](https://user-images.githubusercontent.com/4161791/81196771-12f9bb00-8f85-11ea-9fbf-38a2a756799c.png)
